server/container/jetty/src/main/java/org/apache/james/http/jetty/Configuration.java

```diff
@@ -135,6 +135,13 @@ public Builder port(int port) {
             return this;
         }
 
+        public Builder port(Optional<Integer> port) {
```

Comment:
Using your own words : "you should also provide the same method with String [and optional] to easy the call when we know we have some content."

Here I duplicate this method to make the call easier, and avoid boiler plate code when we already have an optional.